src/api/app/assets/stylesheets/webui/bs5-shims.scss

```
@@ -21,3 +21,12 @@
 .fw-light { @extend .font-weight-light; }
 
 .font-monospace { @extend .text-monospace; }
+
+.form-check {
+  @extend .custom-control.custom-radio;
+  @extend .custom-control.custom-checkbox;
+  &.form-switch { @extend .custom-switch; }
+  
+  .form-check-input { @extend .custom-control-input; }
+  .form-check-label { @extend .custom-control-label; }
+}
```

```suggestion
.form-check {
  &.form-switch { @extend .custom-switch; }
  
  .form-check-input { @extend .custom-control-input; }
  .form-check-label { @extend .custom-control-label; }
}
```
Given the errors returned by the feature tests, where the radios are rendered as checkboxes, I guess it would be better to leave the changes from `.custom-control.custom-radio` to `.form-check` and from `.custom-control.custom-checkbox` to `.form-check` to be included in #13602.

---

Awesome. :+1: